src/index.mako.html

```diff
@@ -188,12 +188,10 @@
 % endif
     <div id="loader"></div>
     <div tabindex="1" ga-map ga-map-map="map" ga-map-ol3d="::ol3d">
-      <div ng-cloak translate-cloak id="copyrightsdata">
-        <div>
-          <span >&copy;&nbsp;</span>
-          <span translate>data</span>
-        </div>
-        <div ga-attribution ga-attribution-map="map"></div>
+      <div ng-cloak translate-cloak
+           ga-attribution
+           ga-attribution-map="map"
```

Comment:
I know it's unrelated to this PR but you could use `ga-attribution-map="::map"` here to remove one watcher.